openmeetings-web/src/main/java/org/apache/openmeetings/web/admin/connection/ConnectionsPanel.java

```diff
@@ -45,75 +51,203 @@
+		SearchableDataProvider<ConnectionListItem> sdp = new SearchableDataProvider<>(null) {
 			private static final long serialVersionUID = 1L;
 
-			private List<Client> list() {
-				List<Client> l = new ArrayList<>();
-				l.addAll(cm.list());
-				return l;
-			}
-
 			@Override
-			public Iterator<? extends Client> iterator(long first, long count) {
-				List<Client> l = list();
+			public Iterator<? extends ConnectionListItem> iterator(long first, long count) {
+				List<ConnectionListItem> l = getConnections();
 				return l.subList((int)Math.max(0, first), (int)Math.min(first + count, l.size())).iterator();
 			}
 
 			@Override
 			public long size() {
-				return list().size();
+				return getConnections().size();
 			}
 		};
 		final WebMarkupContainer container = new WebMarkupContainer("container");
 		final WebMarkupContainer details = new WebMarkupContainer("details");
-		SearchableDataView<Client> dataView = new SearchableDataView<>("clientList", sdp) {
+		SearchableDataView<ConnectionListItem> dataView = new SearchableDataView<>("clientList", sdp) {
 			private static final long serialVersionUID = 1L;
 
 			@Override
-			protected void populateItem(final Item<Client> item) {
-				Client c = item.getModelObject();
-				item.add(new Label("type", "html5"));
-				item.add(new Label("login", c.getUser().getLogin()));
-				item.add(new Label("since", getDateFormat().format(c.getConnectedSince())));
-				item.add(new Label("scope", c.getRoom() == null ? "html5" : "" + c.getRoom().getId()));
-				item.add(new Label("server", c.getServerId()));
-				item.add(new BootstrapAjaxLink<String>("kick", null, Buttons.Type.Outline_Danger, new ResourceModel("603")) {
-					private static final long serialVersionUID = 1L;
-					{
-						setSize(Buttons.Size.Small);
-					}
-
-					@Override
-					public void onClick(AjaxRequestTarget target) {
-						cm.invalidate(c.getUserId(), c.getSessionId());
-						target.add(container, details.setVisible(false));
-					}
-				}.add(newOkCancelConfirm(this, getString("605"))));
+			protected void populateItem(final Item<ConnectionListItem> item) {
+				ConnectionListItem connection = item.getModelObject();
+				
+				if (connection.getStream() != null) {
+					ConnectionListKStreamItem kStream = connection.getStream();
+					item.add(new Label("type", kStream.getType()));
+					item.add(new Label("login", kStream.getUid()));
+					item.add(new Label("since", getDateFormat().format(kStream.getConnectedSince())));
+					item.add(new Label("scope", kStream.getStreamType()));
+					item.add(new Label("server", kStream.getSource()));
+					item.add(new BootstrapAjaxLink<String>("kick", null, Buttons.Type.Outline_Danger, new ResourceModel("603")) {
+						private static final long serialVersionUID = 1L;
+						{
+							setSize(Buttons.Size.Small);
+						}
+	
+						@Override
+						public void onClick(AjaxRequestTarget target) {
+							// TODO come up with method to kick off this KStream
```

Comment:
Need to still fix how to kick off this stream. But I'm thinking maybe to add an another PR/Jira for it. Maybe not show this button until functionality is there.